IGI.Lab2/src/CarPark.BLL/Service/AccidentService.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using AutoMapper;
+using CarPark.BLL.Dto;
+using CarPark.DAL.Interfaces;
+
+namespace CarPark.BLL.Service
+{
+    public class AccidentService
```

Comment:
All services should have one interface.
Also, all service methods should be asynchronous.
Refactor that.

Reply:
fixed